### PR TITLE
#143 - added send_session_data option

### PR DESCRIPTION
**Pull request by zhil**

### CODE REVIEW

File: src/EventListener/ExceptionListener.php [hunk 1]
```diff
@@ -54,13 +61,15 @@ public function __construct(
         EventDispatcherInterface $dispatcher,
         RequestStack $requestStack,
         array $skipCapture,
+        $sendSessionData,
```
Comment (Jean85) [inline src/EventListener/ExceptionListener.php]: Please add it to the PHPDoc

File: src/EventListener/ExceptionListener.php [hunk 2]
```diff
@@ -188,4 +203,16 @@ private function setUserValue($user)
             $this->client->set_user_data((string)$user, null, $data);
         }
     }
+
+    private function setSessionData()
+    {
+        if ($this->session) {
+            $this->client->user_context($this->session->all());
```
Comment (Jean85) [inline src/EventListener/ExceptionListener.php]: Can you show me an example of how this data is treated inside Sentry?

Comment (zhil) [reply src/EventListener/ExceptionListener.php]: image

**Comment by Jean85** [comment]
Build seems to be broken due to PHPStan. I'll try to fix it elsewhere.

**Comment by Jean85** [comment]
Triggering rebuild after merging #145...

**Pull request closed without merging**